allure-report-face/src/main/webapp/js/filters.js

```diff
@@ -87,4 +87,24 @@ angular.module('allure.filters', [])
         return function(value) {
             return $sce.trustAsHtml(''+value);
         };
-    }]);
\ No newline at end of file
+    }])
+    .filter('truncate', function () {
+        return function (text, length, end) {
+            if (text === undefined) {
+                return;
+            }
+            if (isNaN(length))
+                length = 122;
```

Comment:
It is just default value which can be overriden (got it from original behaviour). What`s wrong?